client/src/main/java/org/kakara/client/game/world/ClientChunkWriter.java

```diff
@@ -64,26 +54,26 @@ public Chunk getChunkByLocation(ChunkLocation chunkLocation) {
             KakaraGame.LOGGER.error("Unable to get chunk: " + chunkLocation.toString(), e);
```

Comment:
This applies to line # 52. This ods version is outdated, updated to 1.0-Better Memory.